src/GraphQL.Harness/GraphQLMiddleware.cs

```diff
@@ -3,7 +3,6 @@
 using System.Threading.Tasks;
 using GraphQL;
 using GraphQL.Instrumentation;
-using GraphQL.SystemTextJson;
```

Comment:
And the dependency within the middleware upon GraphQL.SystemTextJson has been eliminated.